packages/react-components/src/organisms/UserProfileBackground.tsx

```diff
@@ -1,14 +1,16 @@
 import { useContext } from 'react';
 import { css } from '@emotion/react';
-import { UserResponse, UserTeam } from '@asap-hub/model';
+import { UserResponse, UserTeam, Lab } from '@asap-hub/model';
 import { network, sharedResearch } from '@asap-hub/routing';
 import { UserProfileContext } from '@asap-hub/react-context';
 
 import { Card, Headline2, Headline3, Link } from '../atoms';
 import { mobileScreen, perRem } from '../pixels';
 import UserProfilePlaceholderCard from './UserProfilePlaceholderCard';
+import { UserProfileLabText } from '../molecules';
 
-type UserProfileBackgroundProps = UserTeam & Pick<UserResponse, 'firstName'>;
+type UserProfileBackgroundProps = UserTeam &
+  Pick<UserResponse, 'firstName'> & { labs?: Lab[] };
```

Comment:
Not sure if it is correct to do this, but when we set up the default value, it makes the labs property optional. So we could use the one from `UserResponse`